addons/html_builder/static/src/core/building_blocks/builder_select_item.js

```diff
@@ -32,11 +33,27 @@ export class BuilderSelectItem extends Component {
             getLabel,
         });
         this.state = state;
-        this.onClick = () => {
-            this.env.onSelectItem();
-            operation.commit();
-        };
-        this.onMouseenter = operation.preview;
-        this.onMouseleave = operation.revert;
+        this.operation = operation;
     }
+
+    onClick = () => {
+        this.env.onSelectItem();
+        this.operation.commit();
+    };
+    onKeydown = (ev) => {
+        const hotkey = getActiveHotkey(ev);
+        if (hotkey === "escape") {
+            this.operation.revert();
+            this.removeKeydown?.();
+        }
+    };
```

Comment:
Nitpicking: discard if you want. For the sake of consistency
```suggestion
    onKeydown(ev) {
        const hotkey = getActiveHotkey(ev);
        if (hotkey === "escape") {
            this.operation.revert();
            this.removeKeydown?.();
        }
    };
```